common/src/main/java/org/apache/sedona/common/raster/RasterAccessors.java

```diff
@@ -96,6 +96,31 @@ public static double getWorldCoordY(GridCoverage2D raster, int colX, int rowY) t
         return RasterUtils.getWorldCornerCoordinates(raster, colX, rowY).getY();
     }
 
+    public static String getGeoReference(GridCoverage2D raster) {
+        return getGeoReference(raster, "GDAL");
+    }
+
+    public static String getGeoReference(GridCoverage2D raster, String format) {
+
+        AffineTransform2D affine = RasterUtils.getGDALAffineTransform(raster);
+
+        double scaleX = affine.getScaleX();
+        double skewX = affine.getShearX();
+        double skewY = affine.getShearY();
+        double scaleY = affine.getScaleY();
```

Comment:
Please use inbuilt RasterAccessors functions to get these parameters so that any changes are centralized